test/puppetlabs/http/client/metrics_test.clj

```diff
@@ -71,6 +71,17 @@
       app
       [jetty9/jetty9-service test-metric-web-service]
       {:webserver {:port 10000}}
+      (with-open [client (Async/createClient (doto (ClientOptions.)
```

Comment:
Looking further through this test it seems that `metrics-namespace-test` has similar cross cutting metrics tests, thinking it might make sense to move these there and rename it.